script.js

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
try to format your code a bit. check out the Airbnb code formating that Akira showed u